data_browser/src/HomePage.js

```diff
@@ -36,7 +36,7 @@ function HomePage() {
   }
 
   const links = config.root_paths.map((path) => {
-    return <Link to={viewSingleUrl(path)}>{path}</Link>;
+    return <a href={viewSingleUrl(path)} target="_blank" rel="noreferrer">{path}</a>;
```

Comment:
This is the only change (the other diff will disappear when I merge the other branch)